app/templates/posts.hbs

```diff
@@ -42,6 +44,7 @@
             {{/power-select}}
             {{/unless}}
 
+            {{#unless session.user.isContributor}}
```

Comment:
This is another edge case but if a contributor is collaborating on many drafts with other users who can apply the tags do we want to allow them to filter their post list by tag? 🤔